tests/test_api.py

```diff
@@ -1908,3 +1909,19 @@ def test_do_create_prod_build_branch_required(self, osbs, branch_name):
                                            inner_template=inner_template,
                                            outer_template=outer_template,
                                            customize_conf=customize_conf)
+
+    # osbs is a fixture here
+    @pytest.mark.parametrize(('name', 'data'), [  # noqa
+        ('special-config',
+         {
+             "special.how": "very",
+             "special.type": "charm"
+         }),
+    ])
+    def test_config_map(self, osbs, name, data):
+        assert osbs.create_config_map(name, data)
```

Comment:
All we're asserting about the return value from create_config_map() at the moment is that it's truthy. @lcarva, do we mind whether this is an `HttpResponse` (as currently) or a `ConfigMapResponse`?